packages/pubsub/src/peer-streams.ts

```diff
@@ -102,7 +108,8 @@ export class PeerStreams extends TypedEventEmitter<PeerStreamEvents> {
     this._rawInboundStream = stream
     this.inboundStream = pipe(
       this._rawInboundStream,
-      (source) => lp.decode(source)
+      // (source) => lp.decode(source)
```

Comment:
```suggestion
```

Reply:
Of course, I forgot to get rid of the comment :)